packages/@vuepress/theme-default/components/DropdownLink.vue

```diff
@@ -149,7 +160,8 @@ export default {
 @media (min-width: $MQMobile)
   .dropdown-wrapper
     height 1.8rem
-    &:hover .nav-dropdown
+    &:hover .nav-dropdown,
+    &:focus-within .nav-dropdown
```

Comment:
Opening the submenu on focus-within works ok, but it could be annoying if there's a navbar with many long submenus (i.e. you'd have to tab through every single item to get past the navbar). Adding a [skip navigation link](https://webaim.org/techniques/skipnav/) could help with that (and probably should also be added regardless), but a keyboard user would still have to tab through every submenu item on the way to the one they actually want.

The [Adobe Mega Menu](https://adobe-accessibility.github.io/Accessible-Mega-Menu/) is a good example of the expected keyboard interactions and aria markup:

Reply:
thx for your explanation @missmatsuko

Reply:
@missmatsuko Thank you for comment and review :) You are right, Maybe it can make to focus submenus when key press the enter. Or, can be added focusable property to config file. What do you think about this ?

Reply:
I think there doesn't need to be an tabindex attribute at all. Can the link element be changed to a <button> if it has children submenu items, and an anchor if it has an href?

Reply:
I made testing when ı remove tabindex, element can not focusable in safari and firefox.

Reply:
I made testing when I remove tabindex, element can not focusable in safari and firefox. @missmatsuko